src/test/resources/processor/UploadModelRequestBody.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "all-MiniLM-L6-v2",
-  "version": 1,
+  "name": "traced_small_model",
+  "version": "1.0.0",
   "model_format": "TORCH_SCRIPT",
   "model_task_type": "text_embedding",
   "model_config": {
     "model_type": "bert",
-    "embedding_dimension": 384,
+    "embedding_dimension": 768,
     "framework_type": "sentence_transformers",
     "all_config": "{\"architectures\":[\"BertModel\"],\"max_position_embeddings\":512,\"model_type\":\"bert\",\"num_attention_heads\":12,\"num_hidden_layers\":6}"
   },
-  "url": "https://github.com/opensearch-project/ml-commons/blob/2.x/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_embedding/all-MiniLM-L6-v2_torchscript_sentence-transformer.zip?raw=true"
+  "url": "https://github.com/jmazanec15/k-NN-1/blob/model/traced_small_model.zip?raw=true"
```

Comment:
Temporary: Will update URL to ml-commons once uploaded

Reply:
@jmazanec15 , you can use this one https://github.com/opensearch-project/ml-commons/blob/2.x/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_embedding/traced_small_model.zip?raw=true

Reply:
Cool just switched it.